src/vs/editor/contrib/gotoSymbol/peek/referencesController.ts

```diff
@@ -173,6 +173,18 @@ export abstract class ReferencesController implements editorCommon.IEditorContri
 		});
 	}
 
+	async changeFocusBetweenPreviewAndReferences() {
```

Comment:
`async`?

Reply:
Thanks for your feedback! I will make these changes and commit again.